chia/cmds/db.py

```diff
@@ -34,7 +35,22 @@ def db_upgrade_cmd(ctx: click.Context, no_update_config: bool, **kwargs) -> None
         print(f"FAILED: {e}")
 
 
-if __name__ == "__main__":
-    from chia.util.default_root import DEFAULT_ROOT_PATH
-
-    db_upgrade_func(DEFAULT_ROOT_PATH)
+@db_cmd.command("validate", short_help="validate the (v2) blockchain database. Does not verify proofs")
+@click.option("--db", default=None, type=click.Path(), help="Specifies which database file to validate")
+@click.option(
+    "--validate-blocks",
+    default=False,
+    is_flag=True,
+    help="validate consistency of properties of the encoded blocks and block records",
+)
+@click.pass_context
+def db_validate_cmd(ctx: click.Context, validate_blocks: bool, **kwargs) -> None:
+    try:
+        in_db_path = kwargs.get("input")
```

Comment:
This doesn't match the `--db` argument, so it silently discards the filename given on cmdline.
Discovered when checking manually the incomplete upgrade, and the config was still pointing to the v1 file.
Change to `kwargs.get("db")` or revert the flag to `--input`?

Reply:
thanks for the report! fixed here: https://github.com/Chia-Network/chia-blockchain/pull/10716